lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -122,6 +55,81 @@ def execute(self) -> int:
 
         return errors
 
+    def _clear_remotes(self, group) -> Union[int, list]:
+        errors = 0
+        tools_not_found = []
+        if self.context.remote:
+            remotes = self.context.remote.split(",")
+        else:
+            # We were not given any remotes on the command line, build the list
+            # from the tools group directory.
+            remotes = self.remote(self.tool_group_dir)
+            if (
+                not remotes
+                and group != "default"
+                and self.is_empty(self.tool_group_dir)
+            ):
+                # Unless it is not a default group we will remove any
```

Comment:
Unless it is **not** a default group? And "directories lingering or wasn't cleaned before" seems both agrammatical and a bit redundant. (They're lingering *because* they weren't cleared before...)

Reply:
Ha! yes, I just wrote down as I was thinking, I will update it